primitives/arithmetic/src/traits.rs

```diff
@@ -148,6 +148,20 @@ impl<
 {
 }
 
+/// A meta trait for arithmetic.
+///
+/// Arithmetic types do all the usual stuff you'd expect numbers to do. They are guaranteed to
+/// be able to represent at least `u16` values without loss, hence the trait implies `From<u16>`
```

Comment:
Huh, but it doesn't guarantee signedness of the implementing type, so isn't it more correct to say that you can reprsent at least `i16` without loss instead?

Reply:
It doesn't say that it guarantee unsignedness. The `From<u16>` ensures that the type can represent at least `u16` and that is also what the comment says.

Reply:
Doesn't the fact that it can be converted from a `u16` means that it guarantees unsignedness?